### PR TITLE
Update phpMyAdmin to version 5.2.1

### PR DESCRIPTION
**Pull request by ibennetch**
Signed-off-by: Isaac Bennetch <bennetch@gmail.com>

### CODE REVIEW

**Comment by ibennetch** [comment]
Looks like I have some failing tests, but I'm not really sure what this means.

**Comment by ibennetch** [comment]
I'm not really sure but wonder if this is related to https://github.com/alpinelinux/docker-alpine/issues/226 and https://github.com/alpinelinux/docker-alpine/issues/235 which have caused me issues in the past. We use the alpine image and there seems to be some problem with riscv64 support not being available.

**Review by williamdes — COMMENTED** [review]
👍🏻

File: library/phpmyadmin [hunk 1]
```diff
@@ -3,17 +3,17 @@ Maintainers: Isaac Bennetch <bennetch@gmail.com> (@ibennetch),
              William Desportes <williamdes@wdes.fr> (@williamdes)
 GitRepo: https://github.com/phpmyadmin/docker.git
 
-Tags: 5.2.0-apache, 5.2-apache, 5-apache, apache, 5.2.0, 5.2, 5, latest
-Architectures: amd64, arm32v5, arm32v7, arm64v8, i386, mips64le, ppc64le, s390x
-GitCommit: 71d54acc9309c04ed53a148ca1979b6cdc10c5b9
+Tags: 5.2.1-apache, 5.2-apache, 5-apache, apache, 5.2.1, 5.2, 5, latest
+Architectures:
```
[the remark this answers is not in the document]
Comment (ibennetch) [reply library/phpmyadmin]: Thank you @yosifkit, I'm not sure about what happened there but you seem to have solved it!

**Comment by ibennetch** [comment]
Wow, thanks for the quick merge!

Have a great day.

**Pull request merged**